src/environment.rs

```diff
@@ -247,19 +247,18 @@ impl LockFileUsage {
     }
 }
 
-/// Returns the prefix associated with the given environment. If the prefix
-/// doesn't exist or is not up-to-date it is updated.
+/// Update the prefix if it doesn't exist or if it is not up-to-date.
 ///
 /// The `sparse_repo_data` is used when the lock-file is update. We pass it into
 /// this function to make sure the data is not loaded twice since the repodata
 /// takes up a lot of memory and takes a while to load. If `sparse_repo_data` is
 /// `None` it will be downloaded. If the lock-file is not updated, the
 /// `sparse_repo_data` is ignored.
-pub async fn get_up_to_date_prefix(
+pub async fn update_prefix(
     environment: &Environment<'_>,
     lock_file_usage: LockFileUsage,
     mut no_install: bool,
-) -> miette::Result<Prefix> {
+) -> miette::Result<()> {
```

Comment:
I can imagine that it would still be useful to have the prefix. I don't see a reason to remove it.

Reply:
But that is the whole point of the PR 😅

I am fine with closing it though.

Reply:
I think the name change is still valid though. With a get_ function I would expect a user to also use the return value.

Reply:
I think `update_lockfile` actually makes more sense then 😄

Reply:
Fine by me :D